R/stan.R

```diff
@@ -153,3 +153,50 @@ fit_TIRT_stan <- function(data, init = 0, ...) {
   structure(nlist(fit, data), class = "TIRTfit")
 }
 
+#' @export
```

Comment:
extracts mean components from all parameters but how would we handle the case when we want to whole posterior distribution of the new trait scores not just their means?